scripts/us_bls/jolts/bls_jolts.py

```diff
@@ -59,78 +58,36 @@
-  partial rows so we need to process them manually.
-
-  Args:
-    url: The URL of the resource to fetch
-    sep: The separator between columns 
-    expected_cols: Fetched columns are validated against this list
-
-  Raises:
-    URLError or HTTPError if the data import request fails 
-    Assert failure if columns of fetched CSVs are not as expected
-  """
-  df = None
-  remote_web_page = urllib.request.urlopen(url)
-  assert remote_web_page.getcode() == 200
-
-  for line in remote_web_page:
-    # Split out columns
-    line_text = line.decode("utf-8")
-    line_text = line_text.replace('\r', '')
-    line_text = line_text.replace('\n', '')
-    cols_in_line = [col.strip() for col in line_text.split(sep)]
-
-    # Initialize dataframe with first row
-    if df is None:
-      df = pd.DataFrame(columns = cols_in_line)
-
-    # Otherwise add a new row
-    else:
-      # If a row is missing a trailing column then pad it 
-      cols_in_line.extend((df.shape[1] - len(cols_in_line)) * [""])
-      df = df.append(pd.DataFrame([cols_in_line], columns=df.columns), ignore_index=True)
-
-  assert len(df.columns) == len(expected_cols) and (False not in (df.columns == expected_cols))
-  return df
-
 def generate_cleaned_dataframe():
   """ Fetches and combines BLS Jolts data sources.
 
   Downloads detailed series information from the entire JOLTS dataset.
   Each of the files is read, combined into a single dataframe, and post-processed.
 
   Returns:
-    jolts_df: A dataframe representing the 6 job data categories by industry, year, and seasonal adjustment
-    schema_mapping: A list of tuples that contains information for each data set 
+    jolts_df: A dataframe representing the 6 job data categories by industry, year, and seasonal adjustment.
+    schema_mapping: A list of tuples that contains information for each data set .
   """
   
-  # Download series descriptions (used for seasonally adjusted status and industry code)
+  # Download series descriptions (used for seasonally adjusted status and industry code).
   expected_series_columns = ['series_id','seasonal', 'industry_code', 'region_code', 'dataelement_code', 'ratelevel_code', 'footnote_codes','begin_year', 'begin_period', 'end_year', 'end_period']
-  series_desc = web_to_pandas("https://download.bls.gov/pub/time.series/jt/jt.series", sep="\t", expected_cols=expected_series_columns)
+  series_desc = pd.read_csv("https://download.bls.gov/pub/time.series/jt/jt.series", sep="\s+")
+  assert len(series_desc.columns) == len(expected_series_columns) and False not in (series_desc.columns == expected_series_columns)
   series_desc = series_desc.set_index("series_id")
 
-  # Download various series datapoints -- find more info here https://download.bls.gov/pub/time.series/jt/jt.txt
+  # Download various series datapoints -- find more info here https://download.bls.gov/pub/time.series/jt/jt.txt.
   expected_job_columns = ['series_id', 'year', 'period', 'value', 'footnote_codes']
```

Comment:
Let's move this line closer to its use, e.g., just before the for loop where it is used.